models/repo.go

```diff
@@ -337,6 +344,14 @@ func DeleteRepository(doer *user_model.User, uid, repoID int64) error {
 		}
 	}
 
+	// delete actions artifacts in ObjectStorage after the repo have already been deleted
+	for _, art := range artifacts {
+		if err := storage.ActionsArtifacts.Delete(art.FilePath); err != nil {
```

Comment:
You should not use the file path directly as storage path.

Reply:
I rename art.FilePath to art.StoragePath to identify the field meaning.